packages/office-ui-fabric-react/src/components/Rating/Rating.test.tsx

```diff
@@ -78,6 +78,78 @@ describe('Rating', () => {
     expect((ratingInputs[4] as HTMLInputElement).checked).to.be.eq(true);
   });
 
+  it('Deciamal Rating not aggregate', () => {
+    let exception;
+    let threwException = false;
+    let rating;
+    try {
+      rating = ReactTestUtils.renderIntoDocument<Rating>(
+        <Rating
+          rating={ 2.7 }
+        />
+      );
+    } catch (e) {
+      exception = e;
+      threwException = true;
+    }
+    expect(threwException).to.be.false;
+
+    let renderedDOM = ReactDOM.findDOMNode(rating as React.ReactInstance);
+    let ratingInputs = renderedDOM.querySelectorAll('.ms-Rating-input');
+
+    expect((ratingInputs[0] as HTMLInputElement).checked).to.be.eq(false);
+    expect((ratingInputs[1] as HTMLInputElement).checked).to.be.eq(true);
+    expect((ratingInputs[2] as HTMLInputElement).checked).to.be.eq(false);
+    expect((ratingInputs[3] as HTMLInputElement).checked).to.be.eq(false);
+    expect((ratingInputs[4] as HTMLInputElement).checked).to.be.eq(false);
+
+    let ratingStar = renderedDOM.querySelectorAll('div.ms-Rating-star');
+    const checkHalfState = (ratingToCheck: number, state: boolean) => {
+      let halfStarElements = ratingStar[ratingToCheck - 1].querySelectorAll('.ms-Rating-partialStar');
+      expect(halfStarElements.length === (state ? 1 : 0)).to.be.eq(true);
+    };
+
+    checkHalfState(1, false);
+    checkHalfState(2, false);
+    checkHalfState(3, false);
+    checkHalfState(4, false);
+    checkHalfState(5, false);
+  });
+
+  it('Deciamal Rating - aggregate', () => {
+    let exception;
+    let threwException = false;
+    let rating;
+    try {
+      rating = ReactTestUtils.renderIntoDocument<Rating>(
+        <Rating
+          rating={ 2.7 }
+          aggregate={ true }
```

Comment:
just realized that the comments were lying there.. had to click submit review :/